apps/admin/src/components/custom/LessonAssigner/LessonCard.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Lesson } from "./types.js";
+import { useSortable } from "@dnd-kit/sortable";
+import { Box } from "@adminjs/design-system";
+import React from "react";
+import { CSS } from "@dnd-kit/utilities";
+
+type LessonCardProps = {
+  lesson: Lesson & { columnId: string; id?: string };
+};
+
+export const LessonCard = ({ lesson }: LessonCardProps) => {
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({ id: lesson.id });
+
+  const cardStyle = {
+    transform: CSS.Transform?.toString(transform),
+    transition,
+  };
+
+  return (
+    <Box
+      {...listeners}
+      {...attributes}
+      id={lesson?.id}
+      key={lesson.params.id}
+      flex
+      style={{
+        alignItems: "center",
```

Comment:
I think it would be good to add `cursor: grab` here, and if not difficult - `cursor: grabbing` when dragging.